pyproject.toml

```
@@ -41,7 +41,7 @@ classifiers = [
     "Topic :: Internet :: WWW/HTTP",
 ]
 dependencies = [
-    "starlette>=0.36.3,<0.37.0",
+    "starlette>=0.37.2,<0.38.0",
```

This should be good?
```suggestion
    "starlette>=0.36.3,<0.38.0",
```

---

No, minimal version should NOT be less than `0.37.2`, as it fixes `DeprecationWarning` described in the first post
Also, `0.37.1` fixes another, very important regression: see #11320.